InteractiveHtmlBom/core/ibom.py

```diff
@@ -126,16 +124,8 @@ def natural_sort(l):
 
         extras = []
         if config.extra_fields:
-            if f.ref in extra_data:
-                extras = [extra_data[f.ref].get(ef, '')
-                          for ef in config.extra_fields]
-            else:
-                # Some components are on pcb but not in schematic data.
-                # Show a warning about possibly outdated netlist/xml file.
-                log.warn(
-                        'Component %s is missing from schematic data.' % f.ref)
-                warning_shown = True
```

Comment:
Code related to warning_shown is broken now, should be moved to kicad.py as well.